src/ui/components/button/ChromecastButton.tsx

```diff
@@ -37,7 +37,7 @@ export class ChromecastButton extends PureComponent<CastButtonProps, CastButtonS
   componentDidMount() {
     const player = (this.context as UiContext).player;
     player.addEventListener(PlayerEventType.CAST_EVENT, this.onCastStateChangeEvent);
-    this.setState({ connected: player.cast?.casting });
+    this.setState({ connected: player.cast.chromecast?.casting || false });
```

Comment:
Hmm, `player.cast` is [currently defined](https://docs.theoplayer.com/api-reference/web/theoplayer.chromelessplayer.md#cast) to be `Cast | undefined`. However, in practice, it's *always* defined, even if both Chromecast and AirPlay features are turned off. 🤔 

I should probably fix this in the THEOplayer API types. 😅

Reply:
I changed this to `player.cast.chromecast?.casting ?? false`. Also good?

Reply:
On RN player.cast is always defined.

Reply:
Yes, I know, it's always defined. So the type *should* just be `Cast`, without `| undefined`.

I need to fix that upstream in THEOplayer itself. You're good on your end. 😉